Run_SLF_Files_manually/run_episode_file_1415.R

```diff
@@ -32,25 +32,35 @@ write_temp_to_disk <- TRUE
 
 # Specify TRUE/FALSE for saving the console output to disk
 # Default set as TRUE
-# console_outputs <- TRUE
+console_outputs <- TRUE
 
-#-------------------------------------------------------------------------------
-# save console outputs if `console_outputs == TRUE`
-# if (console_outputs) {
-#   file_name <- stringr::str_glue(
-#     "ep_{year}_console_{format(Sys.time(), '%Y-%m-%d_%H-%M-%S')}.txt"
-#   )
-#   file_path <- get_file_path(
-#     console_output_path(),
-#     file_name,
-#     create = TRUE
-#   )
-#   con <- file(file_path, open = "wt")
-#
-#   # Redirect messages (including warnings and errors) to the file
-#   sink(con, type = "output", split = TRUE)
-#   sink(con, type = "message", append = TRUE)
-# }
+# #-------------------------------------------------------------------------------
+# # save console outputs if `console_outputs == TRUE`
+if (console_outputs) {
+  update <- latest_update()
+
+  con_output_dir <- "/conf/sourcedev/Source_Linkage_File_Updates/_console_output/"
+
+  file_name <- stringr::str_glue(
+    "ep_{year}_{update}_update.txt"
```

Comment:
I like this idea, i think it would be good if we could also incorporate the sys.time in the file name if the file runs again. This would be handy if we needed to compare outputs and check the latest log was successful